plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/util/TupleDomainUtils.java

```diff
@@ -34,7 +50,8 @@ public static boolean areAllFieldsReferenced(TupleDomain<String> tupleDomain, Li
     /**
      * Check if at least one of the provided source field is referenced in the tupleDomain predicates.
      */
-    public static boolean areSomeFieldsReferenced(TupleDomain<String> tupleDomain, List<String> sourceFields) {
+    public static boolean areSomeFieldsReferenced(TupleDomain<String> tupleDomain, List<String> sourceFields)
```

Comment:
anyway to UT the public util methods?

Reply:
Done, added the unit tests to `TestTupleDomainUtilsTest`.